contracts/VotingEscrow.sol

```diff
@@ -87,12 +87,13 @@ contract VotingEscrow is IERC721, IERC721Metadata, IVotes {
     uint internal tokenId;
 
     /// @notice Contract constructor
-    /// @param token_addr `VELO` token address
-    constructor(address token_addr, address art_proxy) {
+    /// @param token_addr `FLOW` token address
+    constructor(address token_addr, address art_proxy, address _owner) {
         token = token_addr;
         voter = msg.sender;
         team = msg.sender;
         artProxy = art_proxy;
+        owner = _owner;
```

Comment:
what is this for?

Reply:
for easier management on nft marketplace